src/citrine/resources/material_template.py

```diff
@@ -115,3 +116,48 @@ class MaterialTemplateCollection(DataConceptsCollection[MaterialTemplate]):
     def get_type(cls) -> Type[MaterialTemplate]:
         """Return the resource type in the collection."""
         return MaterialTemplate
+
+    def get_specs(self, scope: str, id: str, per_page: int = 20) -> Iterator[dict]:
```

Comment:
Same as above. Not a requirement, but I'd suggest the input parameters be similar to the `get` method in data_concepts:
So `scope="id"` by default.